src/style.css

```diff
@@ -1 +1,73 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Better to avoid margins and paddings reset by `*` selector, specify tags instead (not every element have this styles)

https://mateacademy.slack.com/archives/C04EZF03G10/p1674665677527709

You may use this selector for `box-sizing`

Reply:
This comment is not fixed =(